lib/fastlane/actions/install_cocoapods.rb

```diff
@@ -47,6 +47,11 @@ def self.available_options
                                        env_name: "FL_COCOAPODS_ANSI",
                                        description: "Show output with ANSI codes",
                                        default_value: true),
+          FastlaneCore::ConfigItem.new(key: :skip_bundle_exec,
```

Comment:
I'd suggest we use `bundle_exec` instead and have it `true` by default to make it easier :+1: